### DISCUSSION TITLE
list notes vertically instead of tabs?

### DISCUSSION THREAD

**Discussion by bjlockie** [FossifyOrg/General-Discussion#609] [Q&A]
Is there a way to view the notes in a list instead of having tabs for them all?

**Comment by charliefrance** [comment]
Why was this closed? It sounds like a good idea, especially if you have a lot of notes.

**Reply by Aga-C** [reply]
It was closed because there's already a feature request for reworking tabs: https://github.com/FossifyOrg/Notes/issues/27.
